### ISSUE TITLE
Create the host-home program glossary

### ISSUE THREAD

**Issue by c-fuchs** [hackforla/host-home-poc#3]
### Overview
Create a glossary of terms used by SPY for their host-home program. 

### Action Items

- [ ] Review SPY program documents
- [ ] Identify recurring words with their meanings 
- [ ] Add them to glossary

### Resources/Instructions
[SPY program documents](https://drive.google.com/drive/u/0/folders/1PZzzGa8yEPn-wNEibCSDQkP3KikMs73e)
Host-home [glossary]()

**Comment by awadethomas** [comment]
@c-fuchs can @AlexChh93 and I have permission to edit this page?

**Comment by lola3736** [comment]
@ExperimentsInHonesty is this still a valid issue?

**Comment by londondkim** [comment]
> [@ExperimentsInHonesty](https://github.com/ExperimentsInHonesty) is this still a valid issue?

Yes, Bonnie mentioned creating glossary during COP meeting today on 1/24 (https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer). I updated a HUU: Glossary on google sheets at (https://docs.google.com/spreadsheets/d/1C_ITdPFMU2MEAu5Cy09n94v8Oz365Asfl3LWj2R24yE/edit?usp=sharing).

**Comment by lola3736** [comment]
> > [@ExperimentsInHonesty](https://github.com/ExperimentsInHonesty) is this still a valid issue?
> 
> Yes, Bonnie mentioned creating glossary during COP meeting today on 1/24 ([hackforla/product-management#386 (comment)](https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer)). I updated a HUU: Glossary on google sheets at (https://docs.google.com/spreadsheets/d/1C_ITdPFMU2MEAu5Cy09n94v8Oz365Asfl3LWj2R24yE/edit?usp=sharing).

@londondkim is the above google sheet saved on your drive or HUU, I couldn't locate it in HUU.  If not please move this to HUU drive and add label "ready for product lead" for my review, thanks much.

**Comment by londondkim** [comment]
> > > [@ExperimentsInHonesty](https://github.com/ExperimentsInHonesty) is this still a valid issue?
> > 
> > 
> > Yes, Bonnie mentioned creating glossary during COP meeting today on 1/24 ([hackforla/product-management#386 (comment)](https://github.com/hackforla/product-management/issues/386#react-issue-comment-composer)). I updated a HUU: Glossary on google sheets at (https://docs.google.com/spreadsheets/d/1C_ITdPFMU2MEAu5Cy09n94v8Oz365Asfl3LWj2R24yE/edit?usp=sharing).
> 
> [@londondkim](https://github.com/londondkim) is the above google sheet saved on your drive or HUU, I couldn't locate it in HUU. If not please move this to HUU drive and add label "ready for product lead" for my review, thanks much. Also, please put the link in the resource section of this issue.

I'm not sure why, but I don't have access to edit anymore and the file is "view only", owned by Hack for LA.
It is located in 0054 [DRAFT] Glossary folder > Copies of team glossary
Reference photos:

![Image](https://github.com/user-attachments/assets/b26915ac-85b2-479e-b701-1ec252f56090)

![Image](https://github.com/user-attachments/assets/d1ca9965-dac5-4776-b90b-63693efa7f5c)
